apps/okreads-e2e/src/specs/reading-list.spec.ts

```diff
@@ -18,3 +18,47 @@ describe('When: I use the reading list feature', () => {
     );
   });
 });
+
+
+describe('When: I use the want to read to see the snack bar', () => {
+  it('Then: I should see snackbar', async () => {
+    await browser.get('/');
+    await browser.wait(
+      ExpectedConditions.textToBePresentInElement($('tmo-root'), 'okreads')
+    );
+
+    const form = await $('form');
+    const input = await $('input[type="search"]');
+    await input.sendKeys('angular');
+    await form.submit();
+    const items = await $$('[data-testing="book-item"]');
+    expect(items.length).toBeGreaterThan(1);
+    const bookaList = await $('[data-testing="want-to-read-button"]');
+    await bookaList.click();
+  });
+});
+
+describe('When: I use the remove feature list feature and undo the remove on snackbar', () => {
+  it('Then: I should see snackbar and undo', async () => {
+
+
+    const readingListToggle = await $('[data-testing="toggle-reading-list"]');
+    await readingListToggle.click();
+    await browser.wait(
+      ExpectedConditions.textToBePresentInElement(
+        $('[data-testing="reading-list-container"]'),
+        'My Reading List'
+      )
+    );
+    const readinglistItem = await $$('[data-testing="reading-listitem"]');
+    expect(readinglistItem.length).toEqual(1);
+    const removeCircle = await $('[data-testing="remove-book-from-list"]');
+    await removeCircle.click();
+    await browser.executeScript("return await document.querySelector('.mat-simple-snackbar-action')").then( (button : HTMLElement) => {
```

Comment:
This part is tricky, but you can actually do this by accessing the selenium driver directly: `await browser.driver.findElement(by.tagName("simple-snack-bar"));`